app/assets/v2/js/navbar.js

```diff
@@ -32,10 +32,10 @@ const makeMenu = (navbarEl) => {
   };
 
   // pull computedRootStyles from shared.js or compute here
-  const computedRootStyles = (window.hasOwnProperty('computedRootStyles') ? window.computedRootStyles : getComputedStyle(document.documentElement));
+  const computedRootStyles = (Object.prototype.hasOwnProperty.call(window, 'computedRootStyles') ? window.computedRootStyles : getComputedStyle(document.documentElement));
```

Comment:
is this needed? I think this will break the nav as I remember @gdixon doing some changes to window instead because the bundle.

Reply:
@octavioamu i was unable to push cause of eslint pre-commit hook... it kept throwing an error and that was the fix

Reply:
ok now is fixed on stable and master, just fixed the conflicts on this one also.